postgres/init.sql

```diff
@@ -132,7 +132,9 @@ CREATE TABLE NFTs (
   height integer NOT NULL,
   image_hash text NOT NULL,
   block_number integer NOT NULL,
-  minter char(64) NOT NULL
+  minter char(64) NOT NULL,
+  owner char(64) NOT NULL,
+  _cursor integer
```

Comment:
What is the _cursor field for?

Reply:
The field was meant to track at which block a row is inserted. But since the interpretation and logic meant to handle the events will be on the backend, there is no need for the field anymore. I will remove it.